conf/swisnap-init.sh

```diff
@@ -2,107 +2,111 @@
+        yq w -i "${PUBLISHER_APPOPTICS_CONFIG}" v2.publisher.publisher-appoptics.all.endpoint.token "${APPOPTICS_TOKEN}"
+        yq w -i "${PUBLISHER_PROCESSES_CONFIG}" v2.publisher.publisher-processes.all.endpoint.token "${APPOPTICS_TOKEN}"
     fi
 
-    yq d -i ${CONFIG_FILE} log_path
+    yq w -i ${CONFIG_FILE} log_path "${LOG_PATH:-/proc/self/fd/1}"
     yq w -i ${CONFIG_FILE} restapi.addr 0.0.0.0
 
     if [ -n "${LOG_LEVEL}" ]; then
         yq w -i $CONFIG_FILE log_level "${LOG_LEVEL}"
     fi
 
     if [ "${SWISNAP_SECURE}" = true ]; then
-        yq w -i $CONFIG_FILE control.plugin_trust_level 1
-        FLAGS="--keyring-paths ${SWISNAP_HOME}/.gnupg/"
+        yq w -i ${CONFIG_FILE} control.plugin_trust_level 1
+        FLAGS=("--keyring-paths" "${SWISNAP_HOME}/.gnupg/")
     fi
 
     # Use APPOPTICS_HOSTNAME as hostname_alias
     if [ -n "${APPOPTICS_HOSTNAME}" ]; then
-        yq w -i ${CONFIG_FILE} control.plugins.publisher.publisher-appoptics.all.hostname_alias ${APPOPTICS_HOSTNAME}
+        yq w -i "${CONFIG_FILE}" control.plugins.publisher.publisher-appoptics.all.hostname_alias "${APPOPTICS_HOSTNAME}"
     fi
 }
 
 run_plugins_with_default_configs() {
     # Set to true to enable or disable specific plugins
-    PLUGINS_DIR="${SWISNAP_HOME}/etc/plugins.d/"
     if [ "${SWISNAP_ENABLE_APACHE}" = "true" ]; then
-        mv ${PLUGINS_DIR}/apache.yaml.example ${PLUGINS_DIR}/apache.yaml
+        mv "${PLUGINS_DIR}/apache.yaml.example" "${PLUGINS_DIR}/apache.yaml"
     fi
 
     if [ "$SWISNAP_ENABLE_DOCKER" = "true" ]; then
-        mv ${PLUGINS_DIR}/docker.yaml.example ${PLUGINS_DIR}/docker.yaml
-        if [[ -n ${HOST_PROC} ]]; then
-            sed -i 's,procfs: "/proc",procfs: "'${HOST_PROC}'",g' ${PLUGINS_DIR}/docker.yaml
+        mv "${PLUGINS_DIR}/docker.yaml.example" "${PLUGINS_DIR}/docker.yaml"
+        if [[ -n "${HOST_PROC}" ]]; then
+            sed -i 's,procfs: "/proc",procfs: "'${HOST_PROC}'",g' "${PLUGINS_DIR}/docker.yaml"
         fi
     fi
 
     if [ "${SWISNAP_ENABLE_ELASTICSEARCH}" = "true" ]; then
-        mv ${PLUGINS_DIR}/elasticsearch.yaml.example ${PLUGINS_DIR}/elasticsearch.yaml
+        mv "${PLUGINS_DIR}/elasticsearch.yaml.example" "${PLUGINS_DIR}/elasticsearch.yaml"
     fi
 
     if [ "${SWISNAP_ENABLE_KUBERNETES}" = "true" ]; then
-        mv ${PLUGINS_DIR}/kubernetes.yaml.example ${PLUGINS_DIR}/kubernetes.yaml
+        mv "${PLUGINS_DIR}/kubernetes.yaml.example" "${PLUGINS_DIR}/kubernetes.yaml"
     fi
 
     if [ "${SWISNAP_ENABLE_MESOS}" = "true" ]; then
-        mv ${PLUGINS_DIR}/mesos.yaml.example ${PLUGINS_DIR}/mesos.yaml
+        mv "${PLUGINS_DIR}/mesos.yaml.example" "${PLUGINS_DIR}/mesos.yaml"
     fi
 
     if [ "${SWISNAP_ENABLE_MONGODB}" = "true" ]; then
-        mv ${PLUGINS_DIR}/mongodb.yaml.example ${PLUGINS_DIR}/mongodb.yaml
+        mv "${PLUGINS_DIR}/mongodb.yaml.example" "${PLUGINS_DIR}/mongodb.yaml"
```

Comment:
would be good to add NGINX and NGINX PLUS

Reply:
Request moved to backlog.